content/en/05-reference/01-admin/09-tuning-mem-usage.md

```diff
@@ -0,0 +1,59 @@
+
+The component responsible for collecting and storing the telemetry data in your cluster is called a [PEM](/reference/architecture/#vizier-components) (Pixie Edge Module).
+When Pixie is installed, a `vizier-pem` data collector pod is deployed to each node in your cluster via a DaemonSet. By default, `vizier-pem` pods have a 2Gi memory limit (but this can be configured to be lower).
+
+After installing Pixie, it is normal to see an increase in memory usage of the `vizier-pem` pods as they begin storing telemetry data. Once the `vizier-pem`'s memory limit is reached, old telemetry data is expired to make room for new data and memory utilization should not increase any further.
+
+### How does Pixie use memory?
+
+Pixie's `vizier-pem` pods use memory for two main purposes:
+
+- **Collecting telemetry data**: Tracing application traffic, collecting CPU profiles, etc. requires that those those values are stored in memory somewhere as they are processed.
+
+- **Storing telemetry data (short-term)**: After processing, telemetry data is stored in [data tables](/reference/datatables/) in-memory on the nodes in your cluster. Each data table in Pixie (e.g. http_events) has its own maximum size. These tables collect data until the maximum size is reached, and then begin dropping the oldest data. Retention time depends on the level of traffic in your cluster, but will generally be on the order of hours. See the [FAQ](/about-pixie/faq/#data-collection-how-much-data-does-pixie-store) for long-term data storage solutions.
+
+By default, `vizier-pem` pods reserve 60% of their allocated memory for short-term data storage (leaving the other 40% for the collection). For the default 2Gi memory limit, `vizier-pem` pods will reserve 1.2Gi memory for data storage.
+
+For developers with [long-term data storage](/about-pixie/faq/#how-do-i...-how-do-i-export-data-from-the-pixie-platform-import-data), Pixie can be configured to allocate less memory for data storage.
+
+### If Pixie stores data locally, why does it have a cloud component?
+
+Pixie's [cloud](/reference/architecture/#cloud) component serves Pixie's API and UI to make it easy for developers to access telemetry data without being in the same network as the cluster. When you query the Pixie platform using the CLI, Live UI or API, your telemetry data flows through Pixie's Cloud via a reverse proxy as E2E encrypted traffic without any persistence.
+
+## Configuring Pixie's memory usage
+
+The proper setting for Pixie's memory usage is dependent on your cluster / environment.
+
+For most clusters, we recommend that you initially try deploying Pixie with the default (2Gi) memory limit. To accommodate your application pods, we recommend that this amount be no more than 25% of your nodes' total memory. In other words, with the default configuration your nodes should have 4Gi+ total memory.
```

Comment:
`2Gi` (here and other places)

Reply:
2Gi is 50% of 4Gi. So you might just want to say, use a 1Gi memory limit in Pixie if you have 4Gi total on the node.

Reply:
Let's make a bulleted list of the top level takeaways for each of these sections for the skimmers. There can also be prose but I think bulleted with the tl;dr will be useful.